src/screens/AddSiteScreen.js

```diff
@@ -1,200 +1,51 @@
 import React, {Component} from 'react';
```

Comment:
This screen is now pretty bare, since most of the form logic was removed. We'll build this out with the Pending and Submitted sites later. One thing I did add was the top right nav of the +List icon which now kicks off the Add Site logic